### PR TITLE
SAN-3035 Dock Removed

### PR DESCRIPTION
**Pull request by Myztiq**
Added migrating page to detention.

This won't get hit until both of these are deployed:
- https://github.com/CodeNow/link/pull/8
- https://github.com/CodeNow/link/pull/8
- [X] Tested on Gamma

screen shot 2016-01-08 at 3 07 00 pm
# review
- [x] @anandkumarpatel
- [x] @bkendall

### CODE REVIEW

**Comment by anandkumarpatel** [comment]
+1

**Comment by bkendall** [comment]
:+1:

File: app.js [hunk 1]
```diff
@@ -152,6 +152,15 @@ app._processNaviError = function (req, res, next) {
     res.status(503);
     log.trace('processNaviError type not_running');
 
+    if(keypather.get(req.instance, 'contextVersion.attrs.dockRemoved')){
```
Comment (rsandor) [inline app.js]: spacing around parens and curlies?

**Comment by bkendall** [comment]
I'm taking back my check mark. :) I think you can write a test for this!

**Comment by bkendall** [comment]
GASP! I think writing that test actually found a bug! tests++

**Comment by bkendall** [comment]
LGTM!

**Pull request merged**